applications/MeshMovingApplication/custom_processes/impose_mesh_motion_process.cpp

```diff
@@ -14,6 +14,7 @@
 #include "impose_mesh_motion_process.h"
 
 // Project includes
+#include "containers/model.h"
```

Comment:
this should be in the header as it is used there

More detailed explanation:
The header includes `model_part.h` which has a [forward declaration for `Model`](https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/includes/model_part.h#L66-L67) to avoid circular inlcudes

=> thats why it compiles but it is fragile IMO, so better move it to the header

Reply:
Got it, I thought it was deliberately encouraged to include it only in sources in order to reduce compilation time.

In this case though, wouldn't it make sense to post-include ```model.h``` in directly in ```model_part.h```?